qt/scientific_interfaces/Indirect/ISISEnergyTransfer.cpp

```diff
@@ -453,40 +451,36 @@ ISISEnergyTransfer::createMapFile(const QString &groupType) {
     if (groupFile == "")
       emit showMessageBox("You must enter a path to the .map file.");
 
-    return qMakePair(QString("File"), groupFile);
+    return std::make_pair("File", groupFile.toStdString());
   } else if (groupType == "Groups") {
-    QString groupWS = "Grouping";
-    createDetectorGroupingWorkspace(groupWS);
-    return qMakePair(QString("Workspace"), groupWS);
+    return std::make_pair("Custom", createDetectorGroupingString());
   } else if (groupType == "Default")
-      return qMakePair(QString("IPF"), QString());
+      return std::make_pair("IPF", "");
     else if (groupType == "Custom")
-      return qMakePair(QString("Custom"), m_uiForm.leCustomGroups->text());
+      return std::make_pair("Custom", m_uiForm.leCustomGroups->text().toStdString());
     else {
     // Catch All and Individual
     return qMakePair(groupType, QString());
```

Comment:
I get a compilation error here about conversion between QPair and std::pair here.